lib/parameters_strapi.py

```diff
@@ -0,0 +1,273 @@
+###
+### Copyright (C) 2024 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+import copy
+import itertools
+
+def gen_avc_cqp_strapi_variants(spec, profiles):
+  for case, params in spec.items():
+    variants = copy.deepcopy(params.get("variants", dict()).get("cqp_strapi", None))
```

Comment:
If we consolidate the parameters as described, then it would only be "strapi" variant.  We could also pass the "variant" as a parameter to the generator... then it could be reused for other variants as needed.

Reply:
updated, thanks suggestion